scripts/deploy.bash

```diff
@@ -51,7 +53,7 @@ echo "- gunicorn"
 sudo systemctl restart gunicorn
 
 # invalidate cloudfront
-aws cloudfront create-invalidation --distribution-id $DISTID --invalidation-batch="Paths={Quantity=1,Items=["/*"]},CallerReference=$(date)"
+# aws cloudfront create-invalidation --distribution-id $DISTID --invalidation-batch="Paths={Quantity=1,Items=["/*"]},CallerReference=$(date)"
```

Comment:
will need to revert this (and/or maybe put it in an if statement that tests whether `$DISTID == ""` or not) before deploy

Reply:
ok will add in the `if` statement in bash.